src/base/bittorrent/session.cpp

```diff
@@ -535,7 +538,8 @@ Session::Session(QObject *parent)
     connect(&m_networkManager, &QNetworkConfigurationManager::configurationChanged, this, &Session::networkConfigurationChange);
 
     m_ioThread = new QThread(this);
-    m_resumeDataSavingManager = new ResumeDataSavingManager {m_resumeFolderPath};
+    const QDir resumeDataDir(m_resumeFolderPath);
+    m_resumeDataSavingManager = new ResumeDataSavingManager{resumeDataDir.absoluteFilePath(QLatin1String {Utils::DB::FILENAME})};
```

Comment:
Do you really want to have database in BT_backup folder? Why? Not only it has the inappropriate name (we kept it for compatibility purposes only), a separate folder is not needed at all now.

Reply:
A reason I did it is because we have the lock file there which kinda guarantees that we have permission to read/write in that folder. Which in turn will make exporting the torrents back to the all system more manageable.
I open to suggestions for a new location. Also for a better name for the DB/table.